.github/workflows/golangci-lint.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v5
         with:
-         go-version: "1.22.2"
+         go-version: "1.22.3"
```

Comment:
I didn't realize Subham had already opened a PR for this. I found here https://github.com/rook/rook/pull/14175 that we could revert back to using `1.22` and still get the CVE fix, instead of continuing to manage the .z version. Which do we think  would be preferable moving forward?